packages/chakra-components/src/MenuBar/SubMenu.js

```diff
@@ -2,7 +2,6 @@ import React, { createContext, useState, useRef, useEffect } from "react";
 import { PseudoBox, usePrevious, useColorMode } from "@chakra-ui/core";
 import { useId } from "@reach/auto-id";
 
-import { useMenuBarContext } from "./useMenuBarContext";
```

Comment:
Why was this removed?

Reply:
rebase with refractor-menubar and it's not being used in this file.

Reply:
![image](https://user-images.githubusercontent.com/39694575/79636584-5e932300-8196-11ea-968e-0df5967f3bbd.png)

Still used, Bro. 
Please rebase or merge with you branch again.

Reply:
Done 👍